connector/connect/bin/spark-connect

```diff
@@ -30,8 +30,11 @@ export SPARK_HOME=$FWDIR
 SCALA_BINARY_VER=`grep "scala.binary.version" "${SPARK_HOME}/pom.xml" | head -n1 | awk -F '[<>]' '{print $3}'`
 SCALA_ARG="-Pscala-${SCALA_BINARY_VER}"
 
-# Build the jars needed for spark submit and spark connect
-build/sbt "${SCALA_ARG}" -Phive -Pconnect package
+BUILD="${BUILD:-1}"
```

Comment:
Sorry for a posthoc review. I wonder if we should better pick a different name though. `BUILD` might be too common ..  and that might easily conflict.

Reply:
Ha, that's exactly why I did SCCLASSPATH, but didn't think about this. Let me change it to SCBUILD in a followup.